src/libraries/Microsoft.PowerFx.Core/Glue/SimpleResolver.cs

```diff
@@ -194,5 +200,10 @@ public bool TryLookupEnum(DName name, out NameLookupInfo lookupInfo)
         {
             throw new System.NotImplementedException();
         }
+
+        public virtual void SetGlobalSymbols(IReadOnlyDictionary<string, IGlobalSymbol> globalSymbols = null)
+        {
+            _globalSymbols = globalSymbols ?? throw new ArgumentNullException(nameof(globalSymbols));
+        }
```

Comment:
We don't need a new virtual here. 

Add another ctor (so that's not breaking) and that accepts the dictionary.